### PR TITLE
feat: add homepage(main) for the website

### LINKED ISSUE #7: feat: add homepage(main) for the website
Creating a landing page for the Kubesimplify website.

### PR DESCRIPTION
**Pull request by SuperAayush**
Closes #7

Modified the landing page of the website.

This is how the page looks in full-screen mode :
1/2
Screenshot 2022-05-08 at 2 10 58 AM
2/2
Screenshot 2022-05-08 at 2 11 16 AM

This is how the page looks in inspect mode :

Screenshot 2022-05-08 at 2 14 48 AM

cc @verma-kunal @AvineshTripathi

### CODE REVIEW

File: src/components/HomepageFeatures/index.js [hunk 1]
```diff
@@ -1,64 +1,142 @@
...
+// 1. hp refers to Homepage
+// 2. cont refers to Container
+// 3. p refers to Paragraph
+
+// For Example :-
+
+// (hp_cont_3_btn2_content) refers to "Content present in Button number 2  of Container number 3 on Homepage"
+
+// File for homepage
+
```
Comment (AvineshTripathi) [inline src/components/HomepageFeatures/index.js:14]: FIrstly don't use unnecessary spaces and comments 
- comments for importing is of no use 
- please consise the length of comment. Here is what I thing should be there 

```
//The naming convention used in this file is mentioned below.
// 1. hp refers to Homepage
// 2. cont refers to Container
// 3. p refers to Paragraph
// (hp_cont_3_btn2_content) refers to "Content present in Button number 2  of Container number 3 on Homepage"
```

File: src/components/HomepageFeatures/index.js [hunk 2]
```diff
@@ -1,64 +1,142 @@
...
-  );
-}
+// Imported all the images below
+
+import comments from './assests/Comments.png'
+import reactions from './assests/Reactions.png'
+import computer from './assests/Computer.png'
+import githubActions from './assests/github-actions.png'
+import homepageVid from './assests/Videos.png'
+import homepageBlog from './assests/Blogs.png'
```
Comment (AvineshTripathi) [inline src/components/HomepageFeatures/index.js:28]: We have imported this but not used in the file do we need this

**Review by AvineshTripathi — COMMENTED** [review]
The images shown in the top most part are too big in comparison to the design, is it intensional?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

File: src/components/HomepageFeatures/index.js (hunk 2 above)
Comment (SuperAayush) [reply src/components/HomepageFeatures/index.js]: Really sorry forgot to remove those 2 imports after using them in CSS.

**Comment by SuperAayush** [comment]
> The images shown in the top most part are too big in comparison to the design, is it intensional?
> 
> __ [sema-logo](https://semasoftware.com/gh)  ❓ I have a question

It is according to the design but if you are talking about the inspect mode so the zoom percentage is 100% that is the reason it is looking big.__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by AvineshTripathi** [comment]
In the design the space above the images and below images are more  where as here it is less when I say space I mean space between the navbar and image and space between image and below component also for video content and log content space above and below is more in the design

**Comment by SuperAayush** [comment]
> In the design the space above the images and below images are more where as here it is less when I say space I mean space between the navbar and image and space between image and below component also for video content and log content space above and below is more in the design

I believe that the space thing is not that important because images here have to be absolute and the above space part can be fixed once the navbar is done also the below space part is fine if I am not wrong 🤔. As long as you where the component is ending and starting space height won't affect.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by AvineshTripathi** [comment]
See smaller size image looks better so if we can we should

**Comment by AvineshTripathi** [comment]
any updates @SuperAayush

**Comment by AvineshTripathi** [comment]
LGTM!

p.s. the changes mentioned above would be done in furthur PRs

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by verma-kunal — APPROVED** [review]
LGTM! 👍

**Comment by verma-kunal** [comment]
> LGTM!
> 
> p.s. the changes mentioned above would be done in furthur PRs

Yes sure! We can keep working on the page, after the PR has been merged as well! 
So, thats not an issue

**Comment by verma-kunal** [comment]
Thanks for contributing @SuperAayush & for helping @AvineshTripathi :)

**Pull request merged**